### PR TITLE
Add info text about boostagrams

### PR DESCRIPTION
**Pull request by dnlggr**
Adds the post about boostagrams and links it from the podcast episode pages.

Suggestions for a different blog post cover image more than welcome if you can think of one!

🔗 Direct link to the blog post: https://deploy-preview-56--seetee.netlify.app/blog/2022-01-31-whats-in-a-boostagram

## 📸 

Screenshot 2022-01-24 at 11 59 34

### CODE REVIEW

File: posts/2022-01-24-whats-in-a-boostagram.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+title: "What's in a Boostagram"
+summary: 'Closing the Loop is a podcast about Bitcoin and the entrepreneurs, developers, and thinkers who are pushing its evolution. But on top of that, it is also a playground for us to explore and experiment with the technological and financial possibilities that Bitcoin brings to online publishing.'
+date: '2022-01-24'
+blocktime: '720166'
+author: '@dnlggr'
+linkAuthorTwitter: true
+thumbnail: 'thumbnail.jpeg'
+cover: 'cover.jpeg'
+coverImageCaption: 'Cover photo by <a href="https://unsplash.com/@snowshade">Oleg Laptev</a> on <a href="https://unsplash.com/s/photos/megaphone">Unsplash</a>.'
+unlisted: true
```
Comment (dnlggr) [inline posts/2022-01-24-whats-in-a-boostagram.md:11]: @dergigi let me know if we should add this text as normal blog post or have it unlisted and only make it accessible via direct link. We can also change this at any time.

**Comment by theborakompanioni** [comment]
lgtm. really nice! :rocket:

File: posts/2022-01-24-whats-in-a-boostagram.md (hunk 1 above)
Comment (dergigi) [reply posts/2022-01-24-whats-in-a-boostagram.md:11]: I'd opt for a direct link for now, i.e. `unlisted: true`

**Comment by dergigi** [comment]
Love the cover image! Just read through it again, and will make some slight adjustments.

**Comment by dergigi** [comment]
I really like it, but I can't shake the feeling that it's a little bit off and could be improved by quite a bit with one or two small changes. Maybe I'm just tired, so let me sleep on it.

I think one very short paragraph that answers the question immediately would be good, at the very beginning. Then mention that Closing the Loop supports boostagrams, and go into what is currently the first paragraph: what is Closing the Loop, what is Podcasting 2.0, etc. Something like that. More tomorrow! 😴

File: posts/2022-01-31-whats-in-a-boostagram.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+blocktime: '721166'
+authors:
+  - name: '@dnlggr'
+    link: 'https://twitter.com/@dnlggr'
+  - name: '@dergigi'
+    link: 'https://twitter.com/@dergigi'
+thumbnail: 'thumbnail.jpeg'
+cover: 'cover.jpeg'
+coverImageCaption: 'Cover photo by <a href="https://unsplash.com/@snowshade">Oleg Laptev</a> on <a href="https://unsplash.com/s/photos/megaphone">Unsplash</a>.'
+unlisted: true
```
Comment (dnlggr) [inline posts/2022-01-31-whats-in-a-boostagram.md:14]: Getting myself a [Gigi number](https://en.wikipedia.org/wiki/Erd%C5%91s_number) of two. 2️⃣ 🤞 😅

**Comment by dnlggr** [comment]
Ready for re-review @dergigi!

File: posts/2022-01-31-whats-in-a-boostagram.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+You can think of boosts as clapping: if you especially like a part, you show your enthusiasm by pressing the boost button—as often as you like.
+
+## From Boosts to Boostagrams
+
+When you decide to press the boost button at a certain point of the episode, you have the option of including a short message as well. Such a message is called a _boostagram_.
+
+Currently, people often use one-time boosts to highlight a certain part of an episode that they particularly enjoyed—a funny joke, an interesting insight, or a great quote. In the future, boosts and boostagrams might be used as the base for censorship-resistant discussions of the content that is being broadcast. Time will tell how it all develops.
+
+### Play & Pay
```
Comment (dergigi) [inline posts/2022-01-31-whats-in-a-boostagram.md:35]:
```suggestion
## Play & Pay
```

**Review by dergigi — DISMISSED** [review]
Looks good to me! Great job 💪 

I'd put all headings on the same level, hence the change suggestion, but it's just a nitpick. Looks and reads great!

**Comment by dnlggr** [comment]
Updated ✅

**Review by dergigi — APPROVED** [review]

**Pull request merged**